jcommune-view/jcommune-web-controller/src/test/java/org/jtalks/jcommune/web/controller/TopicControllerTest.java

```diff
@@ -205,7 +205,7 @@ public void createTopicShouldNotPassAndMustShowTopicErrorIfItIsInvalid() throws
         ModelAndView mav = controller.createTopic(getDto(), result, BRANCH_ID);
 
         verify(branchService).get(BRANCH_ID);
-        verify(breadcrumbBuilder).getForumBreadcrumb(branch);
+        verify(breadcrumbBuilder).getNewTopicBreadcrumb(branch);
```

Comment:
Ok, @ynahapetyan would update the test to use attributes? And also add similar tests to the aforementioned controllers?